tests/dragonfly/conftest.py

```diff
@@ -149,8 +151,9 @@ async def async_client(async_pool):
     Return an async client to the default instance with all entries flushed.
     """
     client = aioredis.Redis(connection_pool=async_pool)
+    await client.client_setname("test")
     await client.flushall()
-    return client
+    yield client
```

Comment:
why yield?

Reply:
it does not matter here but in general, yielding is better because then you have a place to release resources if needed (after yield).